# ZMAP: Zebrafish Multi-study Atlas Project

ZMAP (Zebrafish Multi-study Atlas Project) is a harmonized single-cell RNA sequencing reference atlas for zebrafish embryogenesis, enabling robust cross-study comparison of cell identities.
ZMAP integrates eight published datasets comprising over 790,000 cells spanning 15 developmental time windows from gastrulation through larval stages with a curated hierarchical cell ontology from germ layer to high-resolution clusters. 
Through an interactive web portal, you can explore 2D and 3D UMAP embeddings, query genes, access to the consensus marker resources and project your dataset on it rapidly.

---

## Web Portal

- **2D browser**: <https://zmap.pages.dev>
- **3D browser**: <https://zmap-3d.pages.dev/>

---

## Download ZMAP Data

All ZMAP reference data are publicly available as AnnData (`.h5ad`) files.

### Reference atlas files

- **Raw counts**  
  <https://pub-dbadc2c623224cb58d93cfa3b950fef5.r2.dev/h5ad/ZMAP_250402_raw.h5ad>  
  *(3.5 GB; raw counts)*

- **Processed (recommended)**  
  <https://pub-dbadc2c623224cb58d93cfa3b950fef5.r2.dev/h5ad/ZMAP_251209_processed_slim.h5ad>
  *(3.5 GB; raw counts + all annotations)*

- **Processed (full)**  
  <https://pub-dbadc2c623224cb58d93cfa3b950fef5.r2.dev/h5ad/ZMAP_251209_processed.h5ad>
  *(~25 GB; processed counts + annotations + intermediate embeddings and graphs)*

- **Symphony reference**  
  <https://pub-dbadc2c623224cb58d93cfa3b950fef5.r2.dev/h5ad/ZMAP_260103_symphony.h5ad>
